python/cudf/cudf/tests/test_indexing.py

```diff
@@ -123,6 +127,51 @@ def test_series_indexing(i1, i2, i3):
             assert series[i] == a1[i]
 
 
+@pytest.mark.parametrize(
+    "arg",
+    [
+        1,
+        -1,
+        "b",
+        np.int32(1),
+        np.uint32(1),
+        np.int8(1),
+        np.uint8(1),
+        np.int16(1),
+        np.uint16(1),
+        np.int64(1),
+        np.uint64(1),
+    ],
+)
+def test_series_get_item_iloc_defer(arg):
+    # Indexing for non-numeric dtype Index
+    ps = pd.Series([1, 2, 3], index=pd.Index(["a", "b", "c"]))
+    gs = cudf.from_pandas(ps)
+
+    expect = ps[arg]
+    got = gs[arg]
+
+    assert_eq(expect, got)
+
+
+@pytest.mark.parametrize(
+    "dtype",
+    ["int32", "int16", "int8", "int64", "uint32", "uint16", "uint8", "uint64"],
```

Comment:
You can use this: https://github.com/rapidsai/cudf/blob/9617fcd452a66bdeb9c2275216e866b78f997508/python/cudf/cudf/tests/utils.py#L32